swagger/swagger.yml

```diff
@@ -0,0 +1,71 @@
+consumes:
+- application/json
+definitions:
+  account:
+    type: object
+    required:
+      - name
+    properties:
+      id:
```

Comment:
missing title, description and example .

Reply:
Done. Kindly Check.